astropy/io/ascii/cds.py

```diff
@@ -116,7 +128,7 @@ def get_cols(self, lines):
             if match:
                 col = core.Column(name=match.group('name'))
                 col.start = int(re.sub(r'[-\s]', '',
```

Comment:
```suggestion
                col.start = int(
                    re.sub(r'[-\s]', '', match.group('start') or match.group('end'))) - 1  # fmt: skip
```
alternative following discussion in https://github.com/astropy/astropy/pull/13956#discussion_r1012412088